dockerfiles/nginx/proxito.conf

```diff
@@ -46,18 +46,4 @@ server {
         add_header X-Served Proxito-404-Fallback always;
     }
 
-    # Main RTD site for non-doc serving needs
-    location @fallback {
```

Comment:
I suppose we can remove this block from -ops as well.

Reply:
Yea, I believe it should already be removed.

Reply:
https://github.com/readthedocs/readthedocs-ops/pull/631/files#diff-e0738b03ea130e6f85f02d6e53914c3dL11